assignment/scripts/loops-practice.js

```diff
@@ -33,15 +42,28 @@ let stars = ['Polaris', 'Gacrux', 'Formalhaut', 'Rigel', 'Deneb']
 // 2.a. TODO: Write a for of loop to console.log each star in the 'stars' array
 console.log('Some stars:');
 
+for (star of stars) {
+  console.log('Some stars:', star);
+}
 
 // 3. While loops  
 console.log('---- 3. While loop ----');
 
 // 3.a. TODO: Write a while loop to console.log each star in the 'stars' array
-console.log('Some stars using while:');
+console.log('Some stars using while:', stars);
 
+while (stars < stars.length) {
```

Comment:
Whoops! This conditional is a bit nonsensical. How can the array `stars` be "less than" the number `stars.length`? 

JavaScript is of course going to try to press on, so you won't get an error here, but you'll notice if you look in the console that this code never runs. What else do you need here to make it work?